config/routes.rb

```diff
@@ -25,9 +25,9 @@ def draw(routes_name)
   
   resource :user_sessions, only: [:create]
   resources :newsletters, only: [:create]
-  get  "first_edition" => "home#first_edition_projects"
-  get  "second_edition" => "home#second_edition_projects"
-  get  "third_edition" => "home#third_edition_projects"
+  get  "featured_first_edition_projects" => "home#first_edition_projects"
```

Comment:
woah that's long name, how about `first_edition_projects`? also did you make sure we're not using eg `first_edition_path` anywhere else? I'm fairly sure we're using it in home view (index.html)

Reply:
oh yeah... thanks for pointing out.